plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
we do. Config::getContextBaseUrls() returns all context base_url[], and we check $requestedBaseUrl against that array. We just move on (setting the general base url from config) if there's not match there. If there's match, we do nothing, because the $requestBaseUrl is already not an alias. And the canonical already uses it via router->url